Samples/Word-Get-Started-With-Dev-Kit/README.md

```diff
@@ -0,0 +1,106 @@
+# Word Hello World Task Pane Add-in
+
+<img src="./assets/thumbnail.png" width="800" alt="A document with 'Hello World' inserted by the add-in."/>
+
+This is an Word hello-world add-in that simply inserts "Hello World" to canvas.
+
+Word add-ins are integrations built by third parties into Word by using [Word JavaScript API](https://learn.microsoft.com/en-us/office/dev/add-ins/reference/overview/word-add-ins-reference-overview) and [Office Platform capabilities](https://learn.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins).
+
+## How to run this sample
```

Comment:
I made some suggestions in the previous readme and run with extension file that should be reflected here, based on the outcome of those discussions.

Reply:
Done, pls do a recheck.

Reply:
Thanks! Added a couple more changes.